src/index.html

```diff
@@ -12,7 +12,79 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div aria-label="Star holder">
+      <section class="stars stars--0">
```

Comment:
The `section` tag is a semantic tag that represents a standalone section of functionality contained within an HTML document, typically with a heading, that could be independently distributed or syndicated. In this case, it would be more appropriate to use the `div` tag because these sections do not seem to represent standalone sections of functionality.